### ISSUE TITLE
DOC: Update or remove broken links to QE lectures

### ISSUE THREAD

**Issue by oyamad** [QuantEcon/QuantEcon.py#691]
For example: https://github.com/QuantEcon/QuantEcon.py/blob/15aa789198f28969c2e7aea005720168a244188d/quantecon/_lss.py#L4-L7

**Comment by rohinish404** [comment]
https://python.quantecon.org/linear_models.html
is this the correct link?

**Comment by m-gopichand** [comment]
closes #691
[28. Linear State Space Models](https://python.quantecon.org/linear_models.html#id3)

Before 
``` js
https://lectures.quantecon.org/py/linear_models.html 
```

After 
```js
https://python.quantecon.org/linear_models.html
```
